.github/initrd/centos.conf

```diff
@@ -0,0 +1,14 @@
+[Distribution]
+Distribution=centos
+
+[Output]
+ImageId=initrd
+Format=cpio
+ManifestFormat=
+CompressOutput=zstd
+
+[Content]
+BasePackages=conditional
```

Comment:
We don't need `BasePackages=conditional` here, we'll only install a minimal set of packages by default even if this is enabled so we can just drop it